src/slic3r/GUI/Jobs/PrintJob.cpp

```diff
@@ -446,7 +446,7 @@ void PrintJob::process(Ctl &ctl)
             std::string curr_job_id;
             json job_info_j;
             try {
-                job_info_j.parse(job_info);
+                (void)job_info_j.parse(job_info);
```

Comment:
nit: Modern way to do this is `std::ignore`
See: https://en.cppreference.com/w/cpp/utility/tuple/ignore

Reply:
Done, thanks.